lib/mongo.ex

```diff
@@ -1129,7 +1129,29 @@ defmodule Mongo do
 
   e.g. long-hand `query` becomes short-hand `q`, snake case `array_filters`
   becomes `arrayFilters`
+
```

Comment:
Should we make a mention of some of the usual options (upsert & multi) ? I was a bit surprised they couldn't be combined, but it makes sense after some thought.